internal/commands/choice/choice.go

```diff
@@ -0,0 +1,40 @@
+package choice
+
+import (
+	"fmt"
+	"math/rand"
+
+	"github.com/bwmarrin/discordgo"
+	"github.com/pajbot/basecommand"
+	"github.com/pajbot/pajbot2-discord/pkg"
+	"github.com/pajbot/pajbot2-discord/pkg/commands"
+)
+
+var _ pkg.Command = &Command{}
+
+func init() {
+	commands.Register([]string{
+		"$choice",
+	}, New())
+}
+
+type Command struct {
+	basecommand.Command
+}
+
+func New() *Command {
+	return &Command{
+		Command: basecommand.New(),
+	}
+}
+
+func (c *Command) Run(s *discordgo.Session, m *discordgo.MessageCreate, parts []string) pkg.CommandResult {
+	parts = parts[1:]
+	response := fmt.Sprintf("%s, %s", m.Author.Mention(), parts[rand.Intn(len(parts))])
+	s.ChannelMessageSend(m.ChannelID, response)
```

Comment:
Make sure we trim out mentions here. You should not be able to do `$choice @everyone @everyone` or `$choice @pajlada @pajlada`